db/blob/blob_file_reader_test.cc

```diff
@@ -232,10 +322,38 @@ TEST_F(BlobFileReaderTest, CreateReaderAndGetBlob) {
     uint64_t bytes_read = 0;
 
     ASSERT_TRUE(reader
-                    ->GetBlob(read_options, incorrect_key, blob_offset,
-                              blob_size, kNoCompression, &value, &bytes_read)
+                    ->GetBlob(read_options, incorrect_key, blob_offsets[0],
```

Comment:
I think we might want to change the incorrect key to `foo1` so the size matches

Reply:
Done